start/scaling-up/scaling-up.md

```diff
@@ -0,0 +1,70 @@
+[title]: - "Scale Up your Workflow on OSG Connect"
+[TOC]
+
+## Background
+
+Much of OGS's computing power comes from the ability to run a large number 
+of jobs simulateously. Many HTC-friendly workflows involve running the same 
+job multiple times with no or minor changes between them. Here, we will 
+talk about how to submit multiple jobs at once using the `queue` command as 
+well as some important considerations and tips for scaling up your HTCondor jobs. 
+For a hands-on example of these concepts, checkout the [OSG Connect Quickstart]
+(https://support.opensciencegrid.org/support/solutions/articles/5000633410-osg-connect-quickstart). 
+
+## Things to Consider
+
+In order to avoid confusion, let's review some HTCondor terminiology:
+ - When you queue a single submit script using the `condor_submit` command, this is 
+called a **cluster** and can consist of one more individual jobs.
+ - Each individual job within a cluster is called a **process**. Therefore, each 
+*cluster* can be made up of one or more *processees*. Each proceess lands on its 
+own job slot and is handled individually from the other processes in the cluster.
+
+### Handling Output

```

Comment:
Make this handling input + output?  This guide doesn't yet cover how to do something like unique input files / arguments.

Reply:
As below: Added a section on Submitting Multiple Jobs with a Single Script which mentions this option and links to CHTC's doc page on it.